doc/pr2_tutorials/planning/src/doc/controller_configuration.rst

```diff
@@ -96,3 +96,14 @@ The ``FollowJointTrajectory`` or ``GripperCommand`` interfaces on your robot mus
  * /gripper_controller/gripper_action/result
 
 You should also be able to see (using *rostopic info topic_name*) that the topics are published/subscribed to by the controllers on your robot and also by the *move_group* node.
+
+Remapping /joint_states topic
+-----------------------------
+
+When you have a program that uses move group, you need to remap the topic /joint_states to /robot/joint_states, otherwise the feedback of the joints will start from zeros (I don't know the exact reason) hence messing the planning process. To do this remapping you could make a simple launch file for your node as follows ::
```

Comment:
"you **may** need to remap"

> otherwise the feedback of the joints will start from zeros (I don't know the exact reason) hence messing the planning process.

"otherwise MoveIt! won't have feedback from the joints"